framework/PageDialogs/MultiSelectDialog.tsx

```diff
@@ -90,62 +77,15 @@ export function MultiSelectDialog<T extends object>(props: MultiSelectDialogProp
       hasNoBodyWrapper
     >
       <ModalBoxBody style={{ overflow: 'hidden' }}>
-        <Split hasGutter>
-          <SplitItem style={{ fontWeight: 'bold', whiteSpace: 'nowrap' }}>
-            {translations.selectedText}
-          </SplitItem>
-          {view.selectedItems.length > 0 ? (
-            <LabelGroup>
-              {view.selectedItems.map((item, i) => {
-                if (modalColumns && modalColumns.length > 0) {
-                  return (
-                    <Label key={i} onClose={() => view.unselectItem(item)}>
-                      <TableColumnCell
-                        item={item}
-                        column={
-                          modalColumns.find(
-                            (column) => column.card === 'name' || column.list === 'name'
-                          ) ?? modalColumns[0]
-                        }
-                      />
-                    </Label>
-                  );
-                }
-                return <></>;
-              })}
-            </LabelGroup>
-          ) : (
-            <SplitItem style={{ fontStyle: 'italic' }}>{translations.noneSelectedText}</SplitItem>
-          )}
-        </Split>
+        <PageMultiSelectList
+          view={view}
```

Comment:
Passing a reference to view so that the calling component can access selected items in `view.selectedItems`.